corehq/apps/app_manager/models.py

```diff
@@ -2095,6 +2095,19 @@ class DefaultCaseSearchProperty(DocumentSchema):
     default_value = StringProperty()
 
 
+class CaseSearchMultimedia(NavMenuItemMediaMixin):
```

Comment:
What would you think of also including the `DictProperty` for the text in here? I don't feel that strongly, I was just thinking it'd be nice to have two fewer attributes in the `CaseSearch` model.

Reply:
Just added new properties to avoid migration. Was not sure if that is feasible

Reply:
My bias is to do the migration, especially since this ticket isn't under time pressure. There are so many places in HQ where I wish we'd spent the time on a migration instead of leaving code that's a little less desirable.